controllers/util/testUtil.go

```diff
@@ -30,6 +32,25 @@ func New(ctx context.Context, k8sClient client.Client, r reconcile.Reconciler) *
 	}
 }
 
+// RandCustomer updates a customer with a random name and email
```

Comment:
Remove this method and make the conversion explicit, where it is requried.

Reply:
sck